SmartDeviceLink/public/SDLMenuCell.m

```diff
@@ -22,6 +22,14 @@ @interface SDLMenuCell()
 @implementation SDLMenuCell
 
 - (instancetype)initWithTitle:(NSString *)title icon:(nullable SDLArtwork *)icon voiceCommands:(nullable NSArray<NSString *> *)voiceCommands handler:(SDLMenuCellSelectionHandler)handler {
```

Comment:
In the `hash` method, we also need to include the new `secondaryText`, `tertiaryText` and `secondaryArtwork`. Tests should also be added to the `SDLMenuCellSpec` to make sure comparing two cells with those properties set works as expected.